src/index.html

```diff
@@ -3,12 +3,35 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <span class="logo">
+        <a href=""><img src="images/logo.png" alt="logo"></a>
```

Comment:
Remember about proper formatting
```suggestion
        <a href="">
          <img src="images/logo.png" alt="logo">
        </a>
```